.github/workflows/ci.yml

```diff
@@ -180,7 +180,7 @@ jobs:
           AZURE_STORAGE_KEY: ${{ secrets.AZURE_STORAGE_KEY }}
           AZURE_BUCKET_NAME: ${{ secrets.AZURE_BUCKET_NAME }}
           SLACK_WEBHOOK: ${{ secrets.SLACK_WEBHOOK_COMMUNITY }}
-          TELEMETRY_ENDPOINT: oss.edge.df.f5.com:443
+          TELEMETRY_ENDPOINT: ${{ github.ref_type == 'branch' && github.event_name == 'push' && startsWith(github.ref, 'refs/heads/release-') && 'oss-dev.edge.df.f5.com:443' || 'oss.edge.df.f5.com:443' }}
```

Comment:
I think `startsWith(github.ref, 'refs/heads/release-')` should be enough. It's a branch name, so we don't have to check the type. And I think we want to use the `dev` endpoint for PRs against the release branch, not just on a push?

Reply:
No the `dev` endpoint is to create images of NGF when we need to push new images to GCP for longevity tests before release. We check for the `ref_type` to be branch to ensure that the dev endpoint is set for images created for longevity tests. 

I know the branch type becomes `tag` when we tag a release to create release images of NGF, those images need to have prod endpoint set.